### PR TITLE
style: try hcp callouts dark

### PR DESCRIPTION
**Pull request by kendallstrautman**
## 🔗 Relevant links

- [Preview link](https://dev-portal-git-kstry-hcp-callouts-dark-hashicorp.vercel.app/waypoint) 🔎
- [Asana task](https://app.asana.com/0/1203652232454021/1204108582255302) 🎟️
- [Figma file](https://www.figma.com/file/v0Hyqvjdf6QJAvRkdIBV24/Dark-Mode?node-id=513-32631&t=xA88yKRiEIdhirlz-0)

## 🗒️ What

This PR updates the background callout graphic svg to have a transparent background so that the card color can serve as the base. Additionally, it pulls out the linear gradient from the svg for the compact callout and reimplements it with css. 

Screenshot 2023-03-17 at 11 07 42 AM

Screenshot 2023-03-17 at 10 49 20 AM

## 📸 Design Screenshots

Screenshot 2023-03-17 at 10 48 59 AM

## 🧪 Testing

- Check a [product landing page](https://dev-portal-git-kstry-hcp-callouts-dark-hashicorp.vercel.app/waypoint)
- Check an[ install page](https://dev-portal-git-kstry-hcp-callouts-dark-hashicorp.vercel.app/waypoint/downloads)

### CODE REVIEW

File: src/components/try-hcp-callout/components/try-hcp-callout-compact/try-hcp-callout-compact.module.css [hunk 1]
```diff
@@ -29,7 +29,13 @@
 
 .background {
 	composes: backgroundBase;
-	background-image: url('./img/try-hcp-callout-compact-background.svg?url');
+	background-color: var(--token-color-surface-faint);
+	background-image: linear-gradient(
+			to left,
+			var(--token-color-surface-primary),
+			transparent 80%
```
Comment (kendallstrautman) [inline src/components/try-hcp-callout/components/try-hcp-callout-compact/try-hcp-callout-compact.module.css:36]: @kellyschummer this is the gradient we will finesse for the install view 'compact' callout.

Comment (kellyschummer) [reply src/components/try-hcp-callout/components/try-hcp-callout-compact/try-hcp-callout-compact.module.css:36]: 👍 the adjustment you have here looks great!!

Comment (kendallstrautman) [reply src/components/try-hcp-callout/components/try-hcp-callout-compact/try-hcp-callout-compact.module.css:36]: Okay sweet! I'll open this for review then.

**Review by kellyschummer — APPROVED** [review]
🏅Beautiful!! Thank you @kendallstrautman

**Review by BRKalow — APPROVED** [review]
Preview looks great!

**Review by ashleemboyer — CHANGES_REQUESTED** [review]
I'm noticing a visual discrepancy for `TryHcpCalloutCompact` in Swingset compared to `staging`. Can we resolve that here since it affects prod?

On this preview:

CleanShot 2023-03-21 at 20 52 10

On staging preview:

CleanShot 2023-03-21 at 20 51 41

**Comment by kendallstrautman** [comment]
@ashleemboyer the source of the visual discrepancy with staging is due to the wide viewport width on swingset. The linear gradient was removed from the svg directly and applied via CSS in the background container. If you shrink the size of the compact swingset example you will see that linear gradient show up again. Let me know if you have more questions there!

**Comment by ashleemboyer** [comment]
> @ashleemboyer the source of the visual discrepancy with staging is due to the wide viewport width on swingset. The linear gradient was removed from the svg directly and applied via [CSS in the background container.](https://github.com/hashicorp/dev-portal/pull/1753#discussion_r1139390144) If you shrink the size of the compact swingset example you will see that linear gradient show up again. Let me know if you have more questions there!

@kendallstrautman should we be proactive about smoothing the new harsh line with the image in case the component is used in an area that's wider than the image? Looks like it's `326px` wide, which is only `16px` of wiggle room compared to the `310px` wide Sidecar. Or can we set a `max-width` on the card so that its intended usage is clearer? I fear that folks without the context may see this and think it's broken, or it'll come up as a forgotten side effect if we decide to make the Sidecar wider a couple of weeks or months from now.

**Pull request merged**